ci-operator/step-registry/mount-ns-hiding/mount-ns-hiding-ref.yaml

```diff
@@ -0,0 +1,12 @@
+ref:
+  as: mount-ns-hiding
+  from: tests
+  commands: mount-ns-hiding-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  grace_period: 5m0s
+  documentation: |-
+    The step applies the machine configuration to enable (or disable) the
```

Comment:
The underlying script can disable, but this step explicitly only enables the feature.

Reply:
I'd be fine to just say it enables the feature, or if you're feeling fancy you could add a Paramater that actually lets the test author do one or the other...
(see https://docs.ci.openshift.org/docs/architecture/step-registry/#parameters)

Reply:
Great suggestion!